Updating commands documentation (#288)
HPInc/ml-git: docs/mlgit_commands.md

ml-git datasets log dataset-ex
```

> The ```--stat``` and ```--fullstat``` must be used when the user wants to bring more information in the log of each tag of the entity.

Reviewer: ```suggestion
The ```--stat``` and ```--fullstat``` must be used when the user wants to show more information in the logs for each tag of the entity.
```

1. push all blobs to the configured data storage.
2. push all metadata related to the commits to the remote metadata repository.

> The options of this command allow the user to configure its behavior in case of failures. During the process of sending blobs, if it fails, ML-Git tries to send the same blob the amount informed in the ```retry``` option. Even if the sending fails after reaching the limit, the command continues sending the other blobs.

Reviewer: ```suggestion
The options of this command allow the user to configure its behavior in case of failures. During the process of sending blobs, if it fails, ML-Git attempts at most 'N' times to send the same blob again, with 'N' being the value informed in the ```retry``` option. However, even if the upload of that blob fails after reaching said limit, the command will still continue to send the other blobs.
```

> The options of this command allow the user to configure its behavior in case of failures. During the process of sending blobs, if it fails, ML-Git attempts at most 'N' times to send the same blob again, with 'N' being the value informed in the ```retry``` option. However, even if the upload of that blob fails after reaching said limit, the command will still continue to send the other blobs.

Reviewer 2: ```suggestion
The options of this command allow the user to configure its behavior in case of failures. During the process of sending blobs, if it fails, ML-Git attempts 'N' times to send the same blob again, with 'N' being the value informed in the ```retry``` option. However, even if the upload of that blob fails after reaching the said limit, the command will still continue to send the other blobs.
```

Note:

```

> You should only use this command for the flexible mutability option. 

Reviewer 2: ```suggestion
You should only use this command in an entity configured with `flexible` mutability option. 
```

ml-git clone https://git@github.com/mlgit-repository
```

> This command will clone the configuration repository and initialize all necessary configurations for ML-Git to work properly. After this command, you can directly use the checkout command for example, without having to initialize each type of entity.

Reviewer: ```suggestion
This command will clone the configuration repository and setup the initial configuration that is necessary for ML-Git to work properly. After executing this command, you can directly use the checkout command, for example, without having to initialize each type of entity.
```

Output:
```
config:
{'batch_size': 20,
 'cache_path': '',
 'datasets': {'git': 'git@github.com:example/your-mlgit-datasets'},
 'index_path': '',
 'labels': {'git': 'git@github.com:example/your-mlgit-labels'},
 'metadata_path': '',
 'mlgit_conf': 'config.yaml',
 'mlgit_path': '.ml-git',
 'models': {'git': 'git@github.com:example/your-mlgit-models'},
 'object_path': '',
 'push_threads_count': 30,
 'refs_path': '',
 'storages': {'s3': {'mlgit-datasets': {'aws-credentials': {'profile': 'mlgit'},
                                     'region': 'us-east-1'}}},
 'verbose': 'info'}
```

> Use this command if you want to check what configuration ml-git is running with. It is highly likely one will need to change the default configuration to adapt for her needs.

Reviewer: ```suggestion
Use this command if you want to check what configuration ml-git is running with. It is highly likely that a user will need to change the default configuration to adapt ml-git for their needs.
```

  --verbose                       Debug mode
```

ML-Git needs a configured storage to store data from managed artifacts. Currently, ML-Git supports five types of storage (S3, MinIO, Azure, GoogleDrive and SFTP).

> This command allows the user to configure the type of storage he needs. Therefore, some options are specific to some storage types, namely:

Reviewer: ```suggestion
This command allows the user to configure the type of storage they need. Therefore, some options are specific to some storage types, namely:
```
